### PR TITLE
Provide initial setup

### PR DESCRIPTION
**Pull request by cytopia**
Provides the first working version of dockerized Cruise Control

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	git
+
+###
+### Download
+###
+RUN set -eux \
+	&& if [ "${VERSION}" = "latest" ]; then \
+			DATA="$( \
+				curl -sS https://github.com/linkedin/cruise-control/releases \
+				| tac \
```
Comment (danvaida) [inline Dockerfile:21]: Are both needed?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+RUN set -eux \
+	&& VERSION="$( cat /VERSION )" \
+	&& ls -lap /tmp/cruise-control/cruise-control-metrics-reporter/build/libs/ \
+	&& cp /tmp/cruise-control/cruise-control-metrics-reporter/build/libs/cruise-control-metrics-reporter-*.jar /cruise-control-metrics-reporter.jar
+
+
+# -------------------------------------------------------------------------------------------------
+# PRODUCTION IMAGE
+# -------------------------------------------------------------------------------------------------
+FROM openjdk:8-jdk-alpine as production
```
Comment (danvaida) [inline Dockerfile:106]: Using "production" here implies that there might be "staging" or others, no?

File: Dockerfile (hunk 1 above)
Comment (cytopia) [reply Dockerfile:21]: Yes, otherwise it will be reversed: [tac](https://linux.die.net/man/1/tac)

File: Dockerfile (hunk 2 above)
Comment (cytopia) [reply Dockerfile:106]: There's a `builder` stage at the beginning. Didn't know about best-practice naming convention for the final stage in Docker builds. Any suggestions?

Comment (danvaida) [reply Dockerfile:106]: `final`?

**Pull request merged**